src/h5web/visualizations/nexus/utils.ts

```diff
@@ -67,31 +62,35 @@ export function getNxDataGroup(
     return undefined;
   }
 
-  if (isNxDataGroup(entity)) {
-    return entity;
+  if (getAttributeValue(entity, 'NX_class') === 'NXdata') {
+    return entity; // `NXdata` group found
```

Comment:
I now look for the `NX_class` attribute directly instead of looking through all the attributes for a value equal to `NXdata`.